domains/sandun.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Portfolio website of Sandun Madhushan",
+  "repo": "https://github.com/sandunMadhushan/sandunMadhushan.github.io",
+  "owner": {
+    "username": "sandunMadhushan",
+    "email": "sandunhmadushan9330@gmail.com"
+  },
+  "record": {
+    "CNAME": "sandunMadhushan.github.io"
```

`CNAME` record must be lowercased
```suggestion
    "CNAME": "sandunmadhushan.github.io"
```